libmscore/tuplet.cpp

```diff
@@ -502,6 +503,11 @@ void Tuplet::layout()
                   }
 
             qreal y3 = p1.y() + (p2.y() - p1.y()) * .5 - l1 * (_isUp ? 1.0 : -1.0);
+            // numbers should in tablature be shown on the same level as the flags
+            // actually the current behaviour doesn't show them on the same height, should be changed ...
+//            if (staff() && staff()->isTabStaff() && staff()->staffType()->slashStyle()) {
+//                  y3 =  (p1.y() <= p2.y()) ? p1.y() : p2.y() - l1 * (_isUp ? 1.0 : -1.0);
```

Comment:
Here the actual heigth should be calculated, probably in relation to the highest staff line and also adding a possibly given user added distance for note values.